Source/Redemption/Private/RedemptionPlayer.cpp

```diff
@@ -121,9 +121,25 @@ void ARedemptionPlayer::Tick(float DeltaTime)
 {
 	Super::Tick(DeltaTime);
 
+	if (FallNoiseFrames > 0)
+	{
+		FallNoiseFrames--;
+	}
+
+	TickSprint(DeltaTime);
+	TickStamina();
+	TickNoiseReporting();
+}
+
+void ARedemptionPlayer::TickSprint(float DeltaTime)
+{
 	if (bIsSprinting && bCanSprint)
 	{
-		Stamina -= StaminaDepletionRate * DeltaTime;
+		if (CanJump())
+		{
+			Stamina -= StaminaDepletionRate * DeltaTime;
```

Comment:
why is it multiplied?

Reply:
it is your code
the multiplication is to make sure that the FPS will not change how long you can sprint

Reply:
Oh yes lol, sorry i was on mobile just quickly looking at it